_maps/map_files/Delta/delta.dmm

```diff
@@ -3799,7 +3799,7 @@
 "bvc" = (/turf/simulated/floor/plasteel{icon_state = "vault"; dir = 1},/area/security/nuke_storage)
 "bvd" = (/obj/machinery/light/small{dir = 1},/obj/structure/closet/crate,/obj/item/coin/silver,/obj/item/coin/silver,/obj/item/coin/silver,/obj/item/coin/silver,/obj/item/coin/silver,/turf/simulated/floor/plasteel{icon_state = "vault"; dir = 1},/area/security/nuke_storage)
 "bve" = (/obj/machinery/atmospherics/unary/vent_scrubber{on = 1},/obj/machinery/status_display{pixel_y = 32},/turf/simulated/floor/plasteel{icon_state = "vault"; dir = 1},/area/security/nuke_storage)
-"bvf" = (/obj/machinery/light/small{dir = 1},/obj/structure/filingcabinet,/obj/item/folder/documents,/turf/simulated/floor/plasteel{icon_state = "vault"; dir = 1},/area/security/nuke_storage)
+"bvf" = (/obj/machinery/light/small{dir = 1},/obj/structure/closet/fireaxecabinet{step_y = 30},/obj/structure/table/reinforced,/turf/simulated/floor/plasteel{icon_state = "vault"; dir = 1},/area/security/nuke_storage)
```

Comment:
No `step_y` please.

Reply:
Should I use pixel y instead?

Reply:
oh my god no, use pixel_y, step_y will literally break the entire map